lib/lambda_ethereum_consensus/validator/validator_set.ex

```diff
@@ -197,12 +201,36 @@ defmodule LambdaEthereumConsensus.ValidatorSet do
     end
   end
 
+  defp maybe_publish_sync_aggregates(set, slot) do
+    # Sync committee is broadcasted for the next slot, so we take the duties for the correct epoch.
+    epoch = Misc.compute_epoch_at_slot(slot + 1)
+    %{sync_subcommittee_participants: participants} = Duties.misc_data(set.duties, epoch)
```

Comment:
Probably bad naming (misc was from miscellaneous), maybe `shared` or `common`? The idea is to have a place for common cached calculations not associated to a particular slots/duty, they could be recalculated at will (every epoch, every period, etc) and may be a way to avoid recalculating costly operations, for example, the map we were discussing regarding validator indices could be placed here as well

Reply:
related to what misc_data means I already answered [here](https://github.com/lambdaclass/lambda_ethereum_consensus/pull/1284#discussion_r1752397670), but aggree that we can have specific function for the specific calculations of the shared data. I'll add it.

Reply:
Created the sync_subcommittee_participants function!